epoxy-annotations/src/main/java/com/airbnb/epoxy/OnVisibilityStateChanged.java

```diff
@@ -0,0 +1,20 @@
+package com.airbnb.epoxy;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * This can be used to annotate methods inside classes with a {@link ModelView} annotation. Methods
```

Comment:
Seems like we should say what parameters a method with this annotation can have?